great_tables/_gt_data.py

```diff
@@ -390,40 +386,23 @@ def align_from_data(self, data: TblData) -> Self:
         # Get a list of `align` values by translating the column classes
         align: list[str] = []
 
+        align_to_left = {"object", "utf8", "string"}
         for col_class in col_classes:
             # Ensure that `col_class` is lowercase
             col_class = str(col_class).lower()
 
             # Translate the column classes to an alignment value of 'left', 'right', or 'center'
-            if col_class == "character-numeric":
+            if col_class == "character-numeric" or _str_detect(col_class, r"int|uint|float|date"):
```

Comment:
@rich-iannone can you double check?

Reply:
There's no difference in auto-alignment behavior from this PR compared to `main`. So everything looks alright to me.